analysis_file/analysis_file.py

```diff
@@ -0,0 +1,142 @@
+import argparse
+import sys
+import time
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCSVIO
+from core_data_modules.util.consent_utils import ConsentUtils
+from core_data_modules.traced_data.util import FoldTracedData
+
+from lib.analysis_keys import AnalysisKeys
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Generates files for analysis from the cleaned and coded '
+        'show and survey responses')
+    parser.add_argument(
+        'user', help='User launching this program')
+    parser.add_argument(
+        'json_input_path', metavar='json-input-path',
+        help='Path to a coded JSON file, containing a list of serialized '
+        'TracedData objects')
+    parser.add_argument(
+        'json_output_path', metavar='json-output-path',
+        help='Path to a JSON file to write serialized TracedData items to '
+        'after modification by this pipeline stage')
+    parser.add_argument(
+        'csv_by_message_output_path', metavar='csv-by-message-output-path',
+        help='Analysis dataset where messages are the unit for analysis (i.e. '
+        'one message per row)')
+    parser.add_argument(
+        'csv_by_individual_output_path', metavar='csv-by-individual-output-path',
+        help='Analysis dataset where respondents are the unit for analysis '
+        '(i.e. one respondent per row, with all their messages joined into a '
+        'single cell).')
+
+    args = parser.parse_args()
+    user = args.user
+    data_input_path = args.json_input_path
+    json_output_path = args.json_output_path
+    csv_by_message_output_path = args.csv_by_message_output_path
+    csv_by_individual_output_path = args.csv_by_individual_output_path
+
+    # Serializer is currently overflowing
+    # TODO: Investigate/address the cause of this.
+    sys.setrecursionlimit(10000)
+
+    demog_keys = [
+        'gender',
+        'gender_raw',
+        'age',
+        'age_raw',
+        'clan_identity',
+        'clan_identity_raw']
+
+    survey_keys = [
+        'needs_met_yesno',
+        'needs_met_raw',
+        'cash_modality_yesno',
+        'cash_modality_raw',
+        'community_priorities_raw',
```

Comment:
No `community_priorities`?

Reply:
No, just the raw keys as community_priorities was not a yes no question